js/azureIdentityCredentialAdapter.spec.ts

```diff
@@ -0,0 +1,19 @@
+import * as assert from "assert";
+import * as dotenv from "dotenv";
+import { ResourceManagementClient } from "@azure/arm-resources";
+import { AzureIdentityCredentialAdapter } from "./azureIdentityCredentialAdapter ";
+
+dotenv.config();
```

Comment:
Change to: dotenv.config({ path: "../.env" });


Because this will be run from the js dir, the .env is one folder up.